src/utils/getArticleContents.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
```

Comment:
Please add a code comment with the link to the issue ticket to explain what this script is for. Thanks.